dbt/adapters/athena/utils.py

```diff
@@ -9,6 +11,27 @@ def clean_sql_comment(comment: str) -> str:
     return " ".join(line for line in split_and_strip if line)
 
 
+def stringify_table_parameter_value(value: Any) -> str:
+    """Convert any variable to string for Glue Table property."""
+    try:
+        if isinstance(value, (dict, list)):
+            value_str: str = json.dumps(value)
+        else:
+            value_str = str(value)
+    except (TypeError, ValueError) as e:
+        # Handle non-stringifiable objects and non-serializable objects
+        value_str = f"Non-stringifiable object: {type(value)}, Error: {str(e)}"
```

Comment:
Maybe an empty string could be cleaner? Otherwise, we save errors and value type in the table metadata.
WDYT?

Reply:
@roslovets any input on this? It is not clear to me why we want to return the error as function result.

Reply:
> @roslovets any input on this? It is not clear to me why we want to return the error as function result.

I thought that it will be easier to debug, because we do not log anything from utils. But empty string also works as representation of empty property.
There is no perfect solution, so I can do an empty string if you are not convienced.

Reply:
Nice, I will try to use logger. I just need some time to test it in production.

Reply:
Make totally sense.